xl/player/player.py

```diff
@@ -397,7 +397,8 @@ def is_playing(self):
             :returns: whether the player is currently playing
             :rtype: bool
         """
-        return self._engine.get_state() == 'playing'
+        return self._engine.get_state() is not None \
```

Comment:
This doesn't look right to me. [`ExaileGstEngine.get_state`](https://github.com/exaile/exaile/blob/master/xl/player/gst/engine.py#L201) can only return `playing`, `paused`, or `stopped`.